docs/index.html

```diff
@@ -1,47 +1,109 @@
+<body class="is-boxed has-animations lights-off">
+    <div class="body-wrap boxed-container">
+        <header class="site-header">
+            <div class="container">
+                <div class="site-header-inner">
+                    <div class="brand header-brand">
+                        <h1 class="m-0">
+                            <div class="lights-toggle">
+                                <input id="lights-toggle" type="checkbox" name="lights-toggle" class="switch">
+                                <label for="lights-toggle" class="text-xs"><span>Turn me <span
+                                            class="label-text">dark</span></span></label>
+                            </div>
+                        </h1>
+                    </div>
                 </div>
-            </article>
-        </div>
-        <div class="st-right">
-            <article>
-                <h1>Sublime Text</h1>
-                <h2>API Documentation</h2>
-                <a href="#" class="st-button user-docs">Coming Soon</a>
-                <div class="repo-icons">
-                    <a href="//github.com/sublimetext-io/api.sublimetext.io">
-                        <img border="0" alt="Github Repo" src="/assets/github_icon.svg" width="25" height="25">
-                    </a>
+            </div>
+        </header>
+
+        <main>
+
+            <section class="features section">
+                <div class="container">
+                    <div class="features-inner section-inner has-bottom-divider">
+                        <div class="features-wrap">
+                            <div class="feature is-revealing">
+                                <div class="feature-inner">
+                                    <a href="//docs.sublimetext.io" class="feature-icon">
+                                        <img class="asset-light" src="dist/images/sublime_text_1024x1024x32.png"
+                                            alt="Sublime Text Documentation">
+                                        <img class="asset-dark" src="dist/images/sublime_text_1024x1024x32.png"
+                                            alt="Sublime Text Documentation">
+                                    </a>
+                                    <div class="feature-content">
+                                        <a href="//docs.sublimetext.io">
+                                            <h3 class="feature-title mt-0">Unofficial Sublime Text Documentation</h3>
+                                        </a>
+                                        <p class="text-sm mb-0">The Sublime Text Community Documentation is a community
+                                            effort at documenting the <a href="https://sublimetext.com/">Sublime
+                                                Text</a> text editor and accompanying its <a
+                                                href="https://sublimetext.com/docs/3/">official documentation</a>.</p>
```

Comment:
The official docs are linked here so you can ignore my earlier comment reagrding that.